raw/GB/language-tables/Central_Veps.csv

```diff
@@ -184,7 +184,7 @@ GB408,Is there any accusative alignment of flagging?,1,Grünthal 2015: 66,,GB408
 GB409,Is there any ergative alignment of flagging?,0,Grünthal 2015:in passim,,
 GB410,Is there any neutral alignment of flagging?,0,Grünthal 2015:59–120,,
 GB415,Is there a politeness distinction in second person forms?,1,Eva Saar (p.c. 2022),,
-GB421,Is there a preposed complementizer for complements of verbs of thinking and/or knowing?,1,Kettunen, Siro 1935: 104,,GB421
+GB421,Is there a preposed complementizer for complements of verbs of thinking and/or knowing?,1,Kettunen & Siro 1935: 104,,GB421

```

Comment:
I don't think editing that file in the current state - with comma as delimiter but no appropriate quoting - makes much sense. Many rows are linked to multiple examples, thus have a comma in the value of the Example col, many features have a comma in their title.

Reply:
Okay. I  did it like this because it was the smallest diff. 

I'll do another commit with quotes on everything.

Reply:
I  can't revert df5d065b8ef5f8d89c5a87b5271151da2db9c273 . When I do, git tells me that the two concerned files, central_veps and north_saami causes conflicts . 

I'll try and find some solution.

Reply:
Solution found.

I've quoted all fields with "".